packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -104,6 +106,10 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [iframeCoordinatesToWindowCoordinates, onMoveComponent],
   );
 
+  const handleKeyPress = e => {
```

Comment:
```suggestion
  const handleKeyDown = e => {
```
Let's keep it consistent with the usage. Same for the props.